cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -116,8 +116,8 @@ type ClusterConfiguration struct {
 	CertificatesDir string
 
 	// ImageRepository sets the container registry to pull images from.
-	// If empty, `k8s.gcr.io` will be used by default; in case of kubernetes version is a CI build (kubernetes version starts with `ci/`)
-	// `gcr.io/k8s-staging-ci-images` will be used as a default for control plane components and for kube-proxy, while `k8s.gcr.io`
+	// If empty, `registry.k8s.io` will be used by default; in case of kubernetes version is a CI build (kubernetes version starts with `ci/`)
+	// `gcr.io/k8s-staging-ci-images` will be used as a default for control plane components and for kube-proxy, while `registry.k8s.io`
```

Comment:
kubeadm can spin up older clusters, right? is the alias valid for all historical image versions as well?

Reply:
yes, older clusters will work if they set up kubeadm configuration correctly. yes, older image versions as well